### PR TITLE
tweaks

### PR DESCRIPTION
**Pull request by snopoke**
@czue 

cc @biyeun

### CODE REVIEW

File: corehq/apps/hqcase/management/commands/ptop_fast_reindex_xforms.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ def handle(self, *args, **options):
         super(Command, self).handle(*args, **options)
 
     def custom_filter(self, view_row):
-        if 'xmlns' in view_row:
+        if view_row and 'xmlns' in view_row:
```
Comment (snopoke) [inline corehq/apps/hqcase/management/commands/ptop_fast_reindex_xforms.py]: have seen `view_row=None` a few times on prod

File: corehq/pillows/user.py [hunk 2]
```diff
@@ -131,6 +131,6 @@ def change_transport(self, doc_dict):
 
 def add_demo_user_to_user_index():
     send_to_elasticsearch(
-        USER_INDEX,
+        'users',
```
Comment (czue) [inline corehq/pillows/user.py]: this is sneaky

**Comment by czue** [comment]
:+1: after :white_check_mark:

File: corehq/pillows/user.py (hunk 2 above)
Comment (snopoke) [inline corehq/pillows/user.py]: Yea, not a big fan of the way this works currently

**Pull request merged**